presto-postgresql/src/test/java/com/facebook/presto/plugin/postgresql/TestPostgreSqlPlugin.java

```diff
@@ -29,6 +29,6 @@ public void testCreateConnector()
     {
         Plugin plugin = new PostgreSqlPlugin();
         ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
-        factory.create("test", ImmutableMap.of("connection-url", "test"), new TestingConnectorContext());
+        factory.create("test", ImmutableMap.of("connection-url", "test", "connection-load-table-mappings", "false"), new TestingConnectorContext());
```

Comment:
Maybe we could tests for this and `sqlserver` connector as well? What do you think?  Maybe you could define a set of tests in `presto-base-jdbc` and them call them from each connector which support mapping? And from jdbc connectors which do not support that we could have negative tests.